### PR TITLE
Minor changes to Conditional Statements

### PR DESCRIPTION
**Pull request by ibinsigma**

### CODE REVIEW

File: parser.py [hunk 1]
```diff
@@ -15,23 +15,20 @@
 upperCase = False
 
 def read_val(map_dict, key):
-    try:
-        return map_dict[key]
-    except:
-        return ''
+    return map_dict.get(key,'')
```
Comment (Sarath18) [inline parser.py]: We can remove this method entirely and use the dictionary get logic inline.

File: parser.py [hunk 2]
```diff
@@ -15,23 +15,20 @@
...
 
 while line:
     line = line.strip()
     x = line.split(" ")
     key = int(x[len(x)-1])
     if(x[1] == "press"):
-        if key == 66 or key == 50 or key == 62:
+        if key in  [66,50,62] :
```
Comment (Sarath18) [inline parser.py]:
```suggestion
        if key in [66,50,62]:
```

File: parser.py [hunk 3]
```diff
@@ -15,23 +15,20 @@
...
             upperCase = not upperCase
         elif key == 22:
             message = message[:-1]
         else:
             if upperCase:
-                if((key >= 10 and key <= 21) or key == 34 or key == 35 or key == 47 or key == 48 or key == 51 or key == 59 or key == 60 or key == 61):
+                if (key >= 10 and key <= 21) or (key in [34,35,47,48,51,59,60,61]) :
```
Comment (Sarath18) [inline parser.py]:
```suggestion
                if (key >= 10 and key <= 21) or (key in [34,35,47,48,51,59,60,61]):
```

**Review by Sarath18 — COMMENTED** [review]
Thank you for the improvements. Please address these minor changes and should be good to go.

File: parser.py (hunk 1 above)
Comment (ibinsigma) [reply parser.py]: Can you elucidate ?

Comment (Sarath18) [reply parser.py]: I meant we can remove the `read_val` method entirely and call the dictionary `.get` method wherever `read_val` is used.

**Review by Sarath18 — APPROVED** [review]

**Pull request merged**